pkg/gofr/datasource/mongo/mongo.go

```diff
@@ -101,28 +102,36 @@ func (c *Client) Connect() {
 
 // InsertOne inserts a single document into the specified collection.
 func (c *Client) InsertOne(ctx context.Context, collection string, document interface{}) (interface{}, error) {
-	defer c.sendOperationStats(&QueryLog{Query: "insertOne", Collection: collection, Filter: document}, time.Now())
+	tracedCtx, span := c.addTrace(ctx, "insert", collection)
```

Comment:
I think we should not rename `insertOne` to `insert` as it would cause parity with other places as every other place we are still using `One` as suffix.

Reply:
sure will use `insert` only

Reply:
okay, but won't the method name `InsertOne` differ from the `method` being shown in tracer. Also, in-case of `FindOne`, `UpdateOne`, `DeleteOne` we are still using the same, so why only change for `Insert`..

Reply:
Ohh sorry my bad, I wanted to change the name of span to `InsertOne` only. You are right we should use the same naming conventions as we are using for other methods having `One` as suffix.

Changed the span name to `insertOne`.